include/zephyr/sys/util_loops.h

```diff
@@ -1051,6 +1051,388 @@
 				_47, _48, _49, _50, _51, _52, _53, _54, _55, \
 				_56, _57, _58, _59, _60, _61, _62, _63, ...) __VA_ARGS__
 
+#define Z_GET_ARGS_FIRST_0(...)
```

Comment:
I think the implementation is straightforward.
I'd be happy to hear any comments on whether you think the name is preferable.